### DISCUSSION TITLE
Cani play Vega Strike on Windows 11?

### DISCUSSION THREAD

**Discussion by Treachery93** [vegastrike/Vega-Strike-Engine-Source#903] [Q&A] [answered]
I've known about Vegas Strike since the early beginnings of the project. I am only now rediscovering this.

Can i play Vega Strike on Windows 11?

If so, how? Any documentation available?

Thanks

**Comment by royfalk** [comment]
There is support for compiling the game on Windows. 

There is currently a discussion on https://app.gitter.im/#/room/#vegastrike_community:gitter.im regarding Windows 10.  It think it should also work on Windows 11.

**Comment by BenjamenMeyer** [comment]
We will build on what is available for [GitHub Action workers](https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories) which is based on the Windows Server edition. Fortunately, the same binaries will work just fine on Win10/11/12.

So one we get our Windows builds fully working and artifacts uploaded it should just work - well, for x86-64 Windows at least. We won't be supporting 32-bit Windows and there is no Windows/ARM runner.

**Comment by BenjamenMeyer** [comment]
> If so, how? Any documentation available?

The 0.9.x and master branches (0.10.x) have scripts to support building on Windows:
- `script/bootstrap.ps1` to install dependencies
- `script/build.ps1` to do the build
We use these in our [GitHub Actions](https://github.com/vegastrike/Vega-Strike-Engine-Source/blob/master/.github/workflows/Windows-CI.yml). There may be some other aspects that need to be figured out too, but they should generally work.

**Reply by BenjamenMeyer** [reply]
Do be aware that the engine works a little differently than the historic engine (0.6.x and earlier) as we have now renamed by primary binary to `vegastrike-engine.exe` and it requires that the directory for the assets be set as a parameter. I think we also provide `vegastrike.exe` which also uses the old behavior; but it's more to help with transition.

**Comment by stephengtuggy** [comment, marked as answer]
The short answer is Yes, you can. I play Vega Strike on Windows 11 regularly.

We even have an installer now. See https://github.com/vegastrike/Vega-Strike-Engine-Source/releases/tag/v0.9.0-beta1 . Use the Windows exe with the higher 10.x version number (10.0.20348); that's the installer for Windows 11.
